### DISCUSSION TITLE
Get an error when importing create_proxy

### DISCUSSION THREAD

**Discussion by rheanV** [pyodide/pyodide#4474] [Q&A] [answered]
I have a use-case where I create a button from Python using tinyhtml and pyodide. I need to create a proxy to wrap Python function in the javascript function which can be called when the button is clicked. Here is my sample code.
```
from pyodide import create_proxy
await addLibrary('tinyhtml')
def runPython():
    print("you clicked me")
 
function_proxy = create_proxy(runPython)
 
content = frag(h("input", type="button", onclick="function_proxy()"))
fig_html1 =  str(content)
#fig_html1 is used to render the button in dom from the javascript side.
print(fig_html1)
  
``` 
I get the error "ImportError: cannot import name 'create_proxy' from 'pyodide' (/lib/python311.zip/pyodide/_init_.py)" when I run this code. What am I missing?
![c999d7a8-06f1-49c7-8a73-b6f012154a0f](https://github.com/pyodide/pyodide/assets/42920310/b83e9aaa-ec1a-4f33-97eb-a9b5fc77c1d4)

**Comment by hoodmane** [comment, marked as answer]
I think you need
```
from pyodide.ffi import create_proxy
```

**Reply by rheanV** [reply]
That worked. Thank you so much.
